apps/judicial-system/web/src/routes/Shared/Cases/CreateCaseButton.tsx

```diff
@@ -0,0 +1,54 @@
+import { FC, useMemo } from 'react'
+import { useIntl } from 'react-intl'
+
+import { Box } from '@island.is/island-ui/core'
+import * as constants from '@island.is/judicial-system/consts'
+import { capitalize } from '@island.is/judicial-system/formatters'
+import { core } from '@island.is/judicial-system-web/messages'
+import { ContextMenu } from '@island.is/judicial-system-web/src/components'
+import { UserRole } from '@island.is/judicial-system-web/src/graphql/schema'
+
+import { CreateCaseButtonProps } from './Cases'
+
+export const CreateCaseButton: FC<CreateCaseButtonProps> = (props) => {
+  const { user } = props
```

Comment:
Could we fetch the user from the `UserContext` directly, instead of passing it down as props?